samples/sqlservices/src/controllers/mainController.ts

```diff
@@ -199,6 +199,7 @@ export default class MainController implements vscode.Disposable {
 		dropdown.onValueChanged((params) => {
 			vscode.window.showInformationMessage(inputBox2.value);
 			inputBox.value = dropdown.value.toString();
+			console.info('dropdown change ' + dropdown.value.toString());
```

Comment:
Do you need this console logging? Can you remove / make it so that it only shows when verbose flag is on?

Reply:
Ah sorry, noticed this is in the sample code. Never mind :)